render/detailed/node_test.go

```diff
@@ -149,7 +149,7 @@ func TestMakeDetailedHostNode(t *testing.T) {
 				Columns:    detailed.NormalColumns,
 				Connections: []detailed.Connection{
 					{
-						ID:       fmt.Sprintf("%s:%s-%s:%s-%d", fixture.ServerHostNodeID, "", fixture.ClientHostNodeID, "", 80),
+						ID:       fmt.Sprintf("%s-%s-%d", fixture.ServerHostNodeID, "", 80),
```

Comment:
I did consider that, but it's not really much of an abstraction - the *contents* of the ID are far more likely to change than the formatting alone, so any change would entail a change in the function and the call sites. I suppose the compiler gives you some help in complaining about mismatches, so perhaps it's worth for that. Will give it a go.